recommendation_types.markdown

```diff
@@ -42,10 +45,16 @@ recommendation type. This resolution process tries to come up with the most
 stable software stack based on the knowledge base. This resolution process is
 the most expensive and takes into account many aspects of the software.
 
+**NOTE:** The resolution process can take up to 20 minutes depending on the
+requirements and system load.
```

Comment:
Is this the standard duration for all recommendation types? (if that's the case, we could provide a single note for all of them, wdyt?)

Reply:
This is just for the ones stated. `latest` and `security` can be faster.

Reply:
Ok, I see 💯